js/interface.js

```diff
@@ -518,9 +519,33 @@ function toggleAll(el) {
 function updatePaths() {
   if (navStack.length > 1) {
     var breadcrumbsPath = '';
+    var type = '';
+    var idType = '';
+    var dataType = '';
 
     for (var i = 0; i < navStack.length; i++) {
-      breadcrumbsPath += '<span class="bread-link"><a href="#" data-breadcrumb="' + i + '">' + navStack[i].name + '</a></span>';
+      switch (navStack[i].type) {
+        case 'organizationId':
+          type = 'organization';
+          idType = 'data-org-id=';
+          dataType = 'data-type=';
```

Comment:
why do these contain the `=` sign? surely that should be part of the initial template

Reply:
fixed